web/client-api/src/main/java/io/deephaven/web/client/api/barrage/def/ColumnDefinition.java

```diff
@@ -161,10 +134,10 @@ public Column makeJsColumn(int index, Map<Boolean, Map<String, ColumnDefinition>
 
         return makeColumn(index,
                 this,
-                format == null || !format.isNumberFormatColumn() ? null : format.getColumnIndex(),
+                style == null ? null : style.getColumnIndex(),
```

Comment:
I'm suspicious about this, just cause it looks like you're converging two things into one. If the convergence is real, maybe you could propagate this into `makeColumn` and `Column` by removing the extra param.

Reply:
Mostly I'm trying to remove internal state, and work out the external implications later, assuming I got it all correct. Yes, it should be flattened into one thing, but I'm leaving an assert so that if there is any bad usage, we find it rather than just rewrite it.